unit-tests/test-fw-update.py

```diff
@@ -72,6 +74,40 @@ def reset_update_counter( device ):
 
     send_hardware_monitor_command( device, cmd )
 
+def find_image_or_exit( product_name, fw_version ):
+    """
+    Description...
+    :param product_name: the name of the camera
+    :param fw_version: fw version
+    :return: the image file corresponding to product_name and fw_version if exist, otherwise exit
+    """
+    pattern = re.compile(r'^Intel RealSense ((\S+?)(\d+)(\S*))')
+    match = pattern.search(product_name)
+    if not match:
+        log.f(product_name, " is not valid - Didn't find 'Intel RealSense ' in product_name")
+
+    # Finding file containing image for FW update
+    # For example: for 'Intel RealSense L53X Recovery'
+    # Take the 'L53' and try to concatenate it 'X' + postfix and than "XX" + postfix
+    # Until find file or exit if not find.
+    x = ''
+    start_index, end_index = match.span(1)
+
+    for i in range(0, end_index-start_index):
+        pn = product_name[start_index:end_index-i]
+
+        image_name = '(^|/)' + pn + x + "_FW_Image-" + fw_version + ".bin" + '$'
+        image_file = None
+        for image in file.find(repo.root, image_name):
+            image_file = os.path.join(repo.root, image)
+        if image_file:
+            break
+        x = x + 'X'
```

Comment:
`D435i` -> `D435X` -> `D43XX` -> ...
Not good, is it?

Reply:
Why not?

Reply:
ok changed